src/index.html

```diff
@@ -13,47 +13,53 @@
   <header>
     <div class="greeting">
       <h2 class="welcome-message"></h2>
-      <h4 class="total-spent"></h4>
+      <h3 class="total-spent"></h3>
     </div>
-    <button class="plan-trip">Plan A New Trip</button>
     <div class="title">
       <h1 class="site-title">Travel<br />Tracker</h1>
```

Comment:
I think semantically, the <h1> should be on top and descend in order from there.  Other than that, this page looks good!